packages/hash/external-services/docker-compose.yml

```diff
@@ -44,19 +48,46 @@ services:
     ports:
       - "${POSTGRES_PORT}:5432"
     volumes:
-      - ../../../var/external-services/postgres/data:/data
+      - hash-postgres-data:/var/lib/postgresql/data
       - ./postgres/postgresql.conf:/etc/postgresql/postgresql.conf:ro
-      - ./postgres/init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
+      - ./postgres/init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh:ro
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready --username ${POSTGRES_USER}"]
+      interval: 1s
```

Comment:
I think this healthcheck runs indefinitely, though, so a 1s interval seems a little too often if it checks forever. Perhaps 5s or 10s is better? Just means we will wait 5s or 10s on initial start before probing (might be fine if it takes a couple retries to suceed, though)

Reply:
The current check will run the check once per second, each second could take 1 second. After 5 retries, it will fail. Optionally, we could also set a start period, but I don't think it's needed. This way at least it tries to complete earlier because I don't really expect it to take several seconds to start. However, the number of retries could be raised, what do you think?

Reply:
It's more that I think the check will run forever and not just until it's in a healthy. From the docs its seems that it won't stop after the first healthy state check?

Reply:
We concluded off-GitHub that we'll just keep it at 2 seconds for now. The healthcheck is cheap and shouldn't be too much of a problem. Keeping the timeout as-is though